### ISSUE TITLE
paste to REPL causes problems on MIMXRT boards

### ISSUE THREAD

**Issue by jerryneedell** [adafruit/circuitpython#3011]
Just a heads up for now -- I'l try to characterize this better.

when I copy/paste code to the REPL on MIMXRT boards I  have seen the following
on Teensy 41 - trying to past this
```
import adafruit_sdcard
import digitalio
import board
import storage
import sys
# Connect to the card and mount the filesystem.
spi = board.SPI()
```
causes the board to enter the bootloader (red LED starts "breathing" - CIRCUITPY drive disconnects)

on the IMIXRT1010-EVK - a similar large-ish paste causes the board to just hang - REPL remains "connected" but unresponsive. Control C does not work -- have to RESET to get it back.

some copy/paste operations work fine on both boards. I have not come up with a clear picture of what triggers the issue. I'll do more testing and report.

may be a duplicate or expansion of #2515 should I move these comments there and close?

**Comment by jerryneedell** [comment]
better example
on both the teensy 41 ans imxrt1010-evk
```
0123456789
0123456789
0123456789
0123456789
0123456789
```  work OK but adding one line
```
0123456789
0123456789
0123456789
0123456789
0123456789
0123456789
```
causes it to crash -- teensy4.1 enters bootloader - imixrt1010-evk  just hangs

**Comment by hathach** [comment]
maybe related to 
- https://github.com/hathach/tinyusb/issues/423 
- https://github.com/micropython/micropython/issues/6034

**Comment by hathach** [comment]
@jerryneedell I submitted an PR to fix this #3225 , when you have time, please check it out.
